packages/cxl-ui/package.json

```diff
@@ -24,7 +24,8 @@
     "@vaadin/vaadin-themable-mixin": "^1.6.2",
     "headroom.js": "^0.12.0",
     "lit-element": "^2.2.1",
-    "lit-html": "^2.0.0-rc.3"
+    "lit-html": "^2.0.0-rc.3",
+    "normalize-wheel": "https://github.com/anoblet/normalize-wheel#esm"
```

Comment:
If we have to fork this anyway, perhaps we can also drop some of the general compatibility check code from it? If not, that's fine, too, but worth a look.

Reply:
I'm not sure how much I can trim away without there being regressions in certain browsers but I can take a look.

Reply:
Is there a PR we can shoot at normalize-wheel upstream way?

I'm going to publish our current ESM build on our custom registry, but ideally we'd at one point get back to using global NPM for this.

Reply:
I tried to structure it as a PR in the first place. I was unsure if I should use a separate directory under `src` or something else... I was also unsure of what was the best practice for offering CJS/ESM in the same package. Maybe I could leverage the `browser` field in `package.json`?

Reply:
I created a PR here: https://github.com/basilfx/normalize-wheel/pull/3

In the meantime I believe we could rely on this: https://www.npmjs.com/package/normalize-wheel-es

Reply:
> https://www.npmjs.com/package/normalize-wheel-es

Looks like they did the same thing, and never bothered to PR.

Let's see what happens in https://github.com/basilfx/normalize-wheel/pull/3 before we make a decision.

Ideally that PR would be a bit less noisy, or code-unified.. but I'm not so well-versed to understand how to pull it off.